### DISCUSSION TITLE
Add incentives to competitive raiding

### DISCUSSION THREAD

**Discussion by CptGermania** [chromiecraft/chromiecraft#3306] [Suggestions]
I would like to request some sort of recognition to players/guilds that manage to claim server first raid clears.

Guilds have been preparing for months for BWL. By doing so they have substantially supported the testing cycles, helped balance and gave extensive feedback.

Seeing more and more guilds drop out from the race to first, i wonder if a predefined reward would have maybe sparked some competitive drive.

Ofc i know this can lead to some super toxic stuff too. But right now it just feels dead and joyless.

**Comment by Denknichts** [comment]
As a small suggestion: "Server first" achievements like in later expansions/retail. And there is room for more here, for example "titles" like "Onyxia Slayer". If I remember correctly, there are such server first titles in the game (later in the game).

**Comment by Cooties-cc** [comment]
A leaderboard would be a nice addition, but not a speed clear type of thing. They could add to the progression page, the top 5 guilds (or groups) that cleared content 1st next to each bracket (raids/dungeons). Could even link to the CC armory for said guild (or players). Although giving out new custom rewards for clearing a raid seem a bit far.

People that report bugs/issues that are confirmed and fixed do get rewarded via cromie points (iirc), and added to a leaderboard. I think that's reasonable enough reward for putting time in to play at a cutting edge and wanting to help the server. People will also get gear and the BWL achievement when they full clear, which has a timestamp date on it. These are generally used as "bragging rights" to show off to other players. However this is where a leaderboard type thing would be neater since it would show who actually got it 1st if multiple guilds cleared on the same day. 

Another problem that would arise would be timezones. If people outside EU times wanted to be serious about getting a 1st, they are at a disadvantage, without trying to attract EUvNA crap from retail RF races. Timezones are just a major problem for people that like competing in video games that are global. A problem that tries to dissolve into the gutter trash that is speedrunning. You've pointed out that it's an overall terrible environment to play in, more so the fact that this server does indeed have bugs. Bugs that could wager a major factor in clear times. It would eventually be a dumpster fire. 

Overall I don't think there should be extra ingame rewards for clearing content at a cutting edge. However I do agree that giving extra visual recognition in the form of a leaderboard would not be a bad idea. It would be a way to immortalize guilds and players on the server, even if they decided to stop playing.

**Comment by 55Honey** [comment]
A Leaderboard is a thing we'd love to have at some point. However,custom titles and achievements require client modifications, a thing we're not willing to touch.

**Reply by CptGermania** [reply]
Well, similarly to what you do with PVP, there are plenty of not (yet) achiebable titles you could give out. But i totally understand thatnit is work ofc.

**Comment by HasieEST** [comment]
With addition of armory, there are improvements that could be applied, which could tackle the suggestion at hand as well.
https://imgur.com/a/S3wZvDj Here is a screenshot of armory from TrueWoW. From the captured Image I'd like to point out that it has raid completion tracking per character. Raids are divided into 3 groups. 60 70 and 80 for respective raids and when hovering over them, you can see which boss specifically a single character has slain. I haven't personally put in a lot of thought on how it tracks it, but with enough time, I think you should have a proper prototype, that could work. Similar thing could be done for Guilds, but instead of tracking players specifically , it would track guild runs, with majority *20/25 or 8/10* of raid consisting of one guild members. Just a food of thought.

**Reply by CptGermania** [reply]
Cool idea! If you could combine it with a timeline of who cleared what first that would already be pretty cool.

**Comment by Talthos** [comment]
I'd honestly like to see additional rewards for completing raids with _less_ than the maximum number of people.

Like, completing a 10-man raid with just 5 or 7 people, for instance.

**Comment by KaneoHunter** [comment]
In order to avoid unnecessary client or even server modifications, I think it would be dope if we could add a "First 3 teams to clear.." to the website progression page. This could also apply to all new content, such as the first teams to clear 5 mans when they release.

https://www.chromiecraft.com/en/progression/

It could list the guilds that beat it, and the characters involved by name when clicked.

I really like what Method have done here, and it serves as my inspiration for this suggestion:
https://www.method.gg/raid-history/vanilla

**Reply by CptGermania** [reply]
Cool idea. Like a leaderboard to commemorate the effort.
At the same time, i still feel like an ingame reward might be better to get people motivated to compete.
